.gitignore

```diff
@@ -3,3 +3,4 @@ Cargo.lock
 **/*.DS_Store
 .idea
 **/*.bk
+hwloc
```

Comment:
Let's only match at the root with `/hwloc`. I might reasonably add an `hwloc` file or directory deeper in the directory tree someday.

Reply:
Ok, I'm actually removing this so that the fetched source resides in the build dir (and is removed by a cargo clean).